pkg/networkplugin-syncer/main.go

```diff
@@ -45,6 +45,7 @@ func main() {
 		klog.Fatalf("Error starting controller: %v", err)
 	}
 
+	<-stopCh
```

Comment:
Should we call ctl.Stop() here?

Reply:
I think it'd be better if we leave go-routine from Start waiting on the stop channel, and then calling Stop, that way it's transparent and works with the stopCh right away. (and saves one line/possible mistake on usages).

Reply:
But good point

Reply:
Having Start wait on the stopCh is problematic for UTs. I've run into this in other tests.

Reply:
I forgot to call Stop here 😄

Reply:
Yeah I was in doubt about if it could be problematic. Then we need to call it on the main.go before stop. that would be fine too as @sridhargaddam proposed.